### DISCUSSION TITLE
GitHub docs REST link anchors scroll to the wrong place

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#67482] [Other Feature Feedback, Questions, & Ideas]
GitHub Mobile for Android

See https://github.com/github/docs/issues/20872#issuecomment-1723351945

1. Follow a link to https://docs.github.com/en/rest/code-scanning/code-scanning?apiVersion=2022-11-28#upload-an-analysis-as-sarif-data

See
![Screenshot_20230918-085837.png](https://github.com/community/community/assets/2119212/7caa0ee8-822e-4258-8c96-711dd080fbad)

Expect to see the header for the linked section. 

Actual results appear to be content for a different section resulting in embarrassing discussions.

**Comment by stevepopovich** [comment]
Switching to general as this is not specific to the native mobile app, thanks for filing!

**Reply by jsoref** [reply]
@stevepopovich are you sure?

Here's what I get in Google Chrome.app on macOS, which seems right:

<img width="834" alt="image" src="https://github.com/community/community/assets/2119212/5042d075-ed16-4a9e-8d2b-a3edfb4e733d">

I know that it's _possible_ for things to be wonky based on screen resolutions, but I tried at least two (this width and double this width) and they seemed fine. Whereas in GitHub Mobile it does the wrong thing. I just tested by following the link on Google Chrome for Android on the same device and it seemed to do the right thing.

**Reply by stevepopovich** [reply]
I see the misalignment when linking from Slack, messages and notes on mobile. I believe the docs screen is scrolling, then rendering more content, moving the scroll position to the incorrect position. This doesn't have anything to do with the native mobile app, but the docs web app itself. We could totally a file an issue for them, but I don't see this as Mobile specific

**Reply by jsoref** [reply]
Ok, gotcha. I presume someone will bridge this issue to the docs web app team. I don't quite understand why it doesn't happen for me when I directly load it in Google Chrome for Android, but I can definitely believe that it would have happened if I tried to follow the link from Slack.

**Reply by stevepopovich** [reply]
yeah, something with the deeplinking is weird there, for sure.
